optuna/storages/_journal/file.py

```diff
@@ -171,6 +171,9 @@ class JournalFileStorage(BaseJournalLogStorage):
 
         lock_obj:
             Lock object for process exclusivity.
```

Comment:
How about moving the newly added explanation to the section of `lock_obj`?

Reply:
Agreed. I revised the docs.